### ISSUE TITLE
Problem using coc-lua on Windows

### ISSUE THREAD

**Issue by tversteeg** [josa42/coc-lua#35]
Hi, I just installed your plugin on Windows 10 using `:CocInstall coc-lua`, everything seems fine but whenever I open a Lua file I get the following error:
```[coc.nvim] Error on active extension coc-lua: Error: ENOENT: no such file or directory, mkdir 'C:\Users\thomas.versteeg\AppData\Local\coc\extensions\coc-lua-data\tools\bin'```

How can I fix this? This is the NeoVim version I'm using:

```
NVIM v0.3.8
Build type: RelWithDebInfo
LuaJIT 2.0.5
Compilation: C:/Program Files (x86)/Microsoft Visual Studio/2017/Community/VC/Tools/MSVC/14.16.27023/bin/Hostx86/x64/cl.exe /DWIN32 /D_WINDOWS /W3 /MD /Zi /O2 /Ob1 /DNDEBUG -DMIN_LOG_LEVEL=3 /W3 -D_CRT_SECURE_NO_WARNINGS -D_CRT_NONSTDC_NO_DEPRECATE -DWIN32 -D_WIN32_WINNT=0x0600 -DINCLUDE_GENERATED_DECLARATIONS -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -IC:/projects/neovim/build/config -IC:/projects/neovim/src -IC:/projects/neovim/.deps/usr/include -IC:/projects/neovim/build/src/nvim/auto -IC:/projects/neovim/build/include
Compiled by appveyor@APPVYR-WIN

Features: -acl +iconv -jemalloc +tui
```

**Comment by tversteeg** [comment]
I see that you marked the question with "help wanted". Is there anything I could do to help or ease the process?

**Comment by josa42** [comment]
@tversteeg Well, I don't have a windows installation, so I have no way to reproduce this issue.

**Comment by tversteeg** [comment]
I can't get `yarn build` to work, but it looks like `mkdir` should be `mkdir -p`?

**Comment by ekspek** [comment]
I just installed this fresh for neovim in WSL (Ubuntu) and I've been getting the same problem.

The printout from `:CocInfo` gives me this:
```
## versions

vim version: NVIM v0.5.0-dev
node version: v8.10.0
coc.nvim version: 0.0.74-bcdd8cdfca
term: st-256color
platform: linux

## Messages
[coc.nvim] Error on active extension coc-lua: Error: ENOENT: no such file or directory, mkdir '/root/.config/coc/extensions/coc-lua-data/tools/bin'
    at Object.fs.mkdirSync (fs.js:885:18)
    at Promise (/root/.config/coc/extensions/node_modules/coc-lua/lib/utils/config.js:19:26)
    at new Promise (<anonymous>)
    at Object.<anonymous> (/root/.config/coc/extensions/node_modules/coc-lua/lib/utils/config.js:18:16)
    at Generator.next (<anonymous>)
    at /root/.config/coc/extensions/node_modules/coc-lua/node_modules/tslib/tslib.js:113:75
    at new Promise (<anonymous>)
    at Object.__awaiter (/root/.config/coc/extensions/node_modules/coc-lua/node_modules/tslib/tslib.js:109:16)
    at Object.configDir (/root/.config/coc/extensions/node_modules/coc-lua/lib/utils/config.js:12:20)
    at Object.<anonymous> (/root/.config/coc/extensions/node_modules/coc-lua/lib/utils/tools.js:21:51)
```

Any ideas?

**Comment by wsw0108** [comment]
@tversteeg @ekspek 

Please make sure below commands works first:
```
PS > mkdir some-dir
PS > luarocks install --tree some-dir --server=http://luarocks.org/dev lua-lsp
```

You need to install luarocks of course.

And I have create a PR #48  to make `coc-lua` use correct binary under Windows(lua-lsp.bat).

**Comment by josa42** [comment]
Should be fixed by #48
